src/pages/locationPage.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+import Card from '../components/Card'
+import location from '../data.json'
+import Header from '../components/Header'
+
+LocationPage.propTypes = {
+  title: PropTypes.string,
+}
+
+export default function LocationPage({ title }) {
```

Comment:
You don't use this title anywhere....
```suggestion
export default function LocationPage() {
```